main.go

```diff
@@ -84,6 +84,7 @@ func main() {
 				configuration.GetPostgresSSLMode(),
 			))
 		if err != nil {
+			db.Close()
```

Comment:
Yes, but I agree that technically it can return nil. So, if you really want I can add one more condition to check for nil.